HomeWork_6/FiveWords.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Class contains method to identify if line has five word
+ * and method that prints result.
+ *
+ * @author Nikita Zenchik
+ */
+public class FiveWords {
+    /**
+     * Method takes String line and finds if it has more then five words.
+     *
+     * @param inputline string line.
+     * @return True if line contains five or less words .
+     * @return False if line contains more then five words.
+     */
+    public static boolean howManyWords(String inputline) {
+        int words = inputline.split("\\w+").length;
+        if (words > 5) {
+            FiveWords.printResult();
```

Comment:
Why you decide to print it here? What if caller want to check the phrase but doesn't want to have any mess in console?